src/main/kotlin/com/cognifide/gradle/aem/tooling/tasks/Tail.kt

```diff
@@ -0,0 +1,96 @@
+package com.cognifide.gradle.aem.tooling.tasks
+
+import com.cognifide.gradle.aem.common.AemDefaultTask
+import com.cognifide.gradle.aem.common.AemTask
+import com.cognifide.gradle.aem.instance.Instance
+import com.cognifide.gradle.aem.tooling.tail.Log
+import com.cognifide.gradle.aem.tooling.tail.LogDestination
+import com.cognifide.gradle.aem.tooling.tail.LogSource
+import com.cognifide.gradle.aem.tooling.tail.Tailer
+import java.io.BufferedReader
+import java.io.File
+import java.io.FileWriter
+import java.io.InputStreamReader
+import java.net.HttpURLConnection
+import java.net.URL
+import java.util.*
+import kotlinx.coroutines.*
+import org.gradle.api.tasks.TaskAction
+import org.gradle.process.internal.shutdown.ShutdownHooks
+
+open class Tail : AemDefaultTask() {
+
+    class AemLogSource(private val aemInstance: Instance) : LogSource {
+
+        override fun nextReader(): BufferedReader {
+            val connection = URL("${aemInstance.httpUrl}$ERROR_LOG_ENDPOINT").openConnection() as HttpURLConnection
```

Comment:
instance.sync { get}

Or instance.httpBasicUrl

Reply:
This is not changed yet - get Socket Closed exceptions and I need to consult it with @pun-ky .

Reply:
OK, thanks to @pun-ky I improved that code.